src/test/resources/regressions/issues/000124.gobra

```diff
@@ -0,0 +1,11 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+package main
+
+// ##(-I src/test/resources/regressions/issues/issue124/)
+import gobra "github.com/gobra"
```

Comment:
Could you add a second test which tests whether `gobra` is inferred from the import path (i.e. by using `import "github.com/gobra"`)